RuntimeUnityEditor/Windows/ObjectTree/ObjectTreeViewer.cs

```diff
@@ -102,27 +103,29 @@ private void DisplayObjectTreeHelper(GameObject go, int indent, ref int currentC
         {
             currentCount++;
 
-            var needsHeightMeasure = _singleObjectTreeItemHeight == 0;
+            var needsHeightMeasure = _singleObjectTreeItemHeight == 0f;
 
-            var isVisible = currentCount * _singleObjectTreeItemHeight >= _treeScrollPosition.y &&
-                            (currentCount - 1) * _singleObjectTreeItemHeight <= _treeScrollPosition.y + _objectTreeHeight;
+            var isVisible = currentCount * _singleObjectTreeItemHeight + _singleObjectTreeItemMargin >= _treeScrollPosition.y &&
+                            (currentCount - 1) * _singleObjectTreeItemHeight + _singleObjectTreeItemMargin <= _treeScrollPosition.y + _objectTreeHeight;
 
             if (isVisible || needsHeightMeasure || _scrollTreeToSelected)
             {
                 if (notVisibleCount > 0)
                 {
-                    GUILayout.Space(_singleObjectTreeItemHeight * notVisibleCount);
+                    GUILayout.Space(_singleObjectTreeItemHeight * notVisibleCount + _singleObjectTreeItemMargin);
                     notVisibleCount = 0;
                 }
 
                 var c = GUI.color;
                 if (SelectedTransform == go.transform)
                 {
                     GUI.color = Color.cyan;
-                    if (_scrollTreeToSelected && Event.current.type == EventType.Repaint)
-                        _scrollTarget = (int)(GUILayoutUtility.GetLastRect().y - 250);
                 }
-                else if (!go.activeSelf)
+                else if (go.scene.name == null && !go.activeInHierarchy)
+                {
+                    GUI.color = new Color(0.6f, 0.6f, 0.4f, 1);
+                }
+                else if (!go.activeInHierarchy)
```

Comment:
Should be using activeInHierarchy here instead of activeSelf IMHO. Makes it much more visible in Object Browser what is currently active and what isn't if a parent is deactivated.